modules/history/src/backend/index.ts

```diff
@@ -0,0 +1,103 @@
+import 'bluebird-global'
+import * as sdk from 'botpress/sdk'
+import _ from 'lodash'
+
+import { Config } from '../config'
+
+const onServerStarted = async (bp: typeof sdk) => {
+  bp.database.createTableIfNotExists('msg_history', table => {
+    table.increments('id').primary()
+    table.date('created_on')
+    table.string('thread_id')
+    table.string('bot_id')
+    table.string('msg_content')
+  })
+}
+
+const cleanDatabase = async (db, limitDate: Date) => {
+  await db
+    .from('msg_history')
+    .where('created_on', '<', limitDate.getTime())
+    .del()
+}
+
+const buildConversationInfo = async (db, threadId: string) => {
+  const messageCountObject = await db
+    .from('msg_history')
+    .count()
+    .where('thread_id', threadId)
+
+  const messageCount = messageCountObject.pop()['count(*)']
+
+  return { id: threadId, count: messageCount }
+}
+
+const onServerReady = async (bp: typeof sdk) => {
+  const router = bp.http.createRouterForBot('history')
+
+  router.get('/conversations/:from/:to', async (req, res) => {
```

Comment:
I missed this one. It's feels just a bit weird to include those params in a route. You might want to use query params instead and make them optional